fuzzers/fuzzer.mk

```diff
@@ -1,12 +1,15 @@
 N ?= 1
 SPECIMENS := $(addprefix build/specimen_,$(shell seq -f '%03.0f' $(N)))
 SPECIMENS_OK := $(addsuffix /OK,$(SPECIMENS))
+ENV_VAR ?=
+FUZDIR ?= ${PWD}
 
 all: database
 
 $(SPECIMENS_OK):
+	echo ${ENV_VAR}
```

Comment:
Remove debug echo?  If you want to keep it, provide context for what is being echo'd